app/javascript/components/mentoring/mentor_inbox/conversation_list/conversation.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(RelativeTime)
+
+export function Conversation({
+  trackTitle,
+  trackIconUrl,
+  menteeAvatarUrl,
+  menteeHandle,
+  exerciseTitle,
+  isStarred,
+  haveMentoredPreviously,
+  isNewIteration,
+  postsCount,
+  updatedAt,
+  url,
+}) {
+  return (
+    <tr>
+      <td>
+        <img
+          style={{ width: 100 }}
+          src={trackIconUrl}
+          alt={`icon indicating ${trackTitle}`}
```

Comment:
Just a small textual suggestion:

```suggestion
          alt={`icon for ${trackTitle} track`}
```